store/store.go

```diff
@@ -164,27 +175,30 @@ func (s *Store[H]) Stop(ctx context.Context) error {
 }
 
 func (s *Store[H]) Height() uint64 {
-	return s.heightSub.Height()
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	head, err := s.Head(ctx)
+	if err != nil {
+		// TODO(cristaloleg): log? panic? retry?
```

Comment:
For the future PR probably.

Reply:
This should definitely be a panic case IMO.

An initialised store should always be able to return a head.